### PR TITLE
refs task #121431 cucumber test for log in

### PR DESCRIPTION
**Pull request by sujoydatta**
pr_sign_in

### CODE REVIEW

File: features/support/env.rb [hunk 1]
```diff
@@ -55,4 +55,6 @@
 # The :transaction strategy is faster, but might give you threading problems.
 # See https://github.com/cucumber/cucumber-rails/blob/master/features/choose_javascript_database_strategy.feature
 Cucumber::Rails::Database.javascript_strategy = :truncation
-
+Capybara.register_driver :selenium do |app|
+  Capybara::Selenium::Driver.new(app, :browser => :chrome)
```
Comment (hellovietnam93) [inline features/support/env.rb]: please check convention here

File: features/step_definitions/common_steps.rb [hunk 2]
```diff
@@ -5,6 +5,18 @@
   visit path_to(page_name)
 end
 
+Given(/^I have (\d+) user with email "([^"]*)" and password "([^"]*)"$/) do
```
Comment (hellovietnam93) [inline features/step_definitions/common_steps.rb]: I think you are confusing in this step. This file is used to define common steps, it means that steps can be reused many times. 
i think all step here is a specific step of a feature. You should put in features/step_definitions/sign_in_steps.rb better

File: features/step_definitions/sign_in_steps.rb [hunk 3]
```diff
@@ -0,0 +1,3 @@
+When(/^I follow "([^"]*)" link$/) do |button|
+  find(button, text: "Sign in").click
+end
```
Comment (hellovietnam93) [inline features/step_definitions/sign_in_steps.rb]: where is Then clause?

Comment (sujoydatta) [inline features/step_definitions/sign_in_steps.rb]: Then clause is in the `common_steps` file because this step is used by other test file too

Comment (hellovietnam93) [inline features/step_definitions/sign_in_steps.rb]: okay :D

**Pull request merged**